soc/xtensa/intel_adsp/ace_v1x/power.c

```diff
@@ -43,7 +46,18 @@ __weak void pm_state_set(enum pm_state state, uint8_t substate_id)
 		if (cpu == 0) {
 			/* FIXME: this value should come from MM */
 			uint32_t hpsram_mask[1] = { 0x3FFFFF };
+#if CONFIG_PM_DEVICE
+			const struct device *dev = device_get_binding("HPSRAM_TLB");
```

Comment:
We are in the process of getting rid of labels in DTS. So it would be great if this is done using `DEVICE_DT_GET(DT_NODELABEL(node_label))`. Also, there is no `HPSRAM_TLB` in the tree right now. So if we enable `CONFIG_PM_DEVICE`, this won't build at all.

Reply:
obsolete, no more PM interface

Reply:
`DEVICE_DT_GET(DT_NODELABEL(tlb))`